src/components/component-placeholder/component-placeholder.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface ComponentPlaceholderProps {
+    name: string;
+    height?: string;
+}
+
+interface ComponentPlaceholderState {
+    componentNumber: number;
+}
+
+class ComponentPlaceholder extends React.Component<ComponentPlaceholderProps, ComponentPlaceholderState> {
```

Comment:
Hva gjør denne komponenten?

Reply:
Lager en visuell placeholder for komponenter som ikke er laget ;-)

![image](https://user-images.githubusercontent.com/29202094/37399774-06a7c76e-2783-11e8-9568-e4231ed8d009.png)